### PR TITLE
Mastering html elements 

### PR DESCRIPTION
**Pull request by MAKWritingHouse**
Mastering html elements

### CODE REVIEW

File: Mastering HTML Elements/Block vs Inline Elements.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <p>p is a block-level paragraph.
+      <!-- You can use block-level inside another block-level or inline element. it will not break the layout ( not recommended and agent the good practice ) -->
+      <h1>h1 is a block-level element</h1>
+      we put one block-level element into another and layout break
+    </p>
+    <!-- ====================
+        Section 1: inline inside block-level element example
+    ===================== -->
+    <p>p is a block-level element.
+      <!-- You can use inline inside another inline or block-level element. it will not break the layout -->
```
Comment (aarif135) [inline Mastering HTML Elements/Block vs Inline Elements.html:25]: Please remove these comments if it is not necessary

**Review by aarif135 — CHANGES_REQUESTED** [review]
Please remove these comments if it is not necessary

File: Mastering HTML Elements/Block vs Inline Elements.html (hunk 1 above)
Comment (MAKWritingHouse) [reply Mastering HTML Elements/Block vs Inline Elements.html:25]: @aarif135 this comments will explain `html tag` and there purpose, kindly allow them if you think this are helpful 😊

**Review by aarif135 — APPROVED** [review]

**Review by makstyle119 — APPROVED** [review]
Mastering html elements

**Pull request merged**